index.js

```diff
@@ -117,11 +117,15 @@ module.exports = {
   treeForApp(app) {
     let trees = [ app ];
 
-    let addonPath = path.join(this.project.root, 'addon');
-    let addonTree = new Funnel(addonPath, {
+    let addonPath;
+    if (this.project.name() === this.name) {
+      addonPath = this.project.root;
+    } else {
+      addonPath = this.project.findAddonByName(this.name).root;
+    }
```

Comment:
I think you can safely just always use the `else` branch here—even in development, `ember-cli-addon-docs` should come up as one of the project's addons

Reply:
Correct. The PR is updated.